obspy/io/sac/tests/test_core.py

```diff
@@ -84,6 +84,12 @@ def test_read_xy_write_xy_via_obspy(self):
             tempfile = tf.name
             tr.write(tempfile, format='SACXY')
             tr1 = read(tempfile)[0]
+
+        sac = tr.stats.pop('sac', None)
+        sac1 = tr1.stats.pop('sac', None)
+        del sac
+        del sac1
```

Comment:
cool, thank you for looking into all this. SAC is used a lot and so it's good if it operates smoothly with obspy.

I do not get the above lines. For me it looks to be the same as:
```python
tr.stats.pop('sac', None)
tr1.stats.pop('sac', None) 
```

What am I missing?